lib/npg_pipeline/analysis/create_lane_tag_file.pm

```diff
@@ -14,8 +14,8 @@ with 'WTSI::DNAP::Utilities::Loggable';
 our $VERSION = '0';
 
 Readonly::Scalar my $TAG_LIST_FILE_HEADER      => qq{barcode_sequence\tbarcode_name\tlibrary_name\tsample_name\tdescription};
-Readonly::Scalar my $SPIKED_PHIX_PADDED        => q{ACAACGCATCTTTCCC};
-Readonly::Scalar my $SPIKED_PHIX_I5OPPOSITE_PADDED => q{ACAACGCAAGATCTCG};
+Readonly::Scalar my $SPIKED_PHIX_PADDED        => q{ACAACGCATCTTTCCC-TCTTTCCC};
```

Comment:
I think base 9 onwards of the i7 is wrong - I'm expecting adapter sequence following the indexing oligo there.